shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/EncryptInsertValuesTokenGenerator.java

```diff
@@ -77,6 +77,9 @@ public InsertValuesToken generateSQLToken(final InsertStatementContext insertSta
     }
     
     private Optional<SQLToken> findPreviousSQLToken(final Class<?> sqlToken) {
+        if (null == previousSQLTokens) {
```

Comment:
Why change the production code?

Reply:
a NPE when previousSQLTokens is null

Reply:
EncryptInsertValuesTokenGenerator need to setPreviousSQLTokens , if not previousSQLTokens will be null.

Reply:
I'm not sure previousSQLTokens  must be set in another place

Reply:
> a NPE when previousSQLTokens is null

previousSQLTokens is Collection, null value is not allowed for a Collection in ShardingSphere's code style.

Reply:
ok, I remove it.